components/server/src/prebuilds/bitbucket-app.ts

```diff
@@ -199,36 +201,30 @@ export class BitbucketApp {
      * @param webhookInstaller the user account known from the webhook installation
      * @returns a promise which resolves to a user account and an optional project.
      */
-    protected async findProjectAndOwner(
-        cloneURL: string,
-        webhookInstaller: User,
-    ): Promise<{ user: User; project?: Project }> {
+    private async findProjectOwner(project: Project, webhookInstaller: User): Promise<User> {
```

Comment:
**praise**: Just wanted to say it feels so good to see this kind of control where 3 repository instances with different prebuilds settings (not enabled, all branches, default branch) work side-by-side like this. Don't know if it's ok, but I only see**one webhook** in the project, see https://gitlab.com/gitpod-io/rails/-/hooks. Cc @AlexTugarev 🙏 

<img width="1342" alt="Screenshot 2023-09-22 at 20 17 29" src="https://github.com/gitpod-io/gitpod/assets/120486/e65d72dc-e82e-4105-98dc-2daa2607bee7">